website/docs/d/contract_subject.html.markdown

```diff
@@ -22,17 +22,27 @@ data "aci_contract_subject" "dev_subject" {
 
 ## Argument Reference
 
-- `contract_dn` - (Required) Distinguished name of parent Contract object.
+- `contract_dn` - (Required) Distinguished name of parent (Contract) object.
 - `name` - (Required) name of Object contract_subject.
 
 ## Attribute Reference
 
 - `id` - Attribute id set to the Dn of the Contract Subject.
-- `annotation` - (Optional) Annotation for object contract subject.
-- `description` - (Optional) Description for object contract subject.
+- `annotation` - (Optional) Annotation for the object contract subject.
+- `description` - (Optional) Description for the object contract subject.
 - `cons_match_t` - (Optional) The subject match criteria across consumers.
-- `name_alias` - (Optional) Name alias for object contract subject.
+- `name_alias` - (Optional) Name alias for the object contract subject.
 - `prio` - (Optional) The priority level of a sub application running behind an endpoint group, such as an Exchange server.
 - `prov_match_t` - (Optional) The subject match criteria across consumers.
 - `rev_flt_ports` - (Optional) Enables filter to apply on ingress and egress traffic.
 - `target_dscp` - (Optional) The target differentiated services code point (DSCP) of the path attached to the layer 3 outside profile.
+
+- `apply_both_directions` - (Optional) . By default set to "yes".
+- ` consumer_to_provider` - (Optional) Filter Chain For Consumer to Provider. Class vzInTerm.
+    - `prio` - (Optional) The priority level of a sub application running behind an endpoint group, such as an Exchange server.
```

Comment:
The priority level to assign to traffic matching the consumer to provider flows.